### ISSUE TITLE
Update Python file headers

### ISSUE THREAD

**Issue by ctb** [dib-lab/khmer#482]
Currently,
```
#                                                                               
# This file is part of khmer, http://github.com/ged-lab/khmer/, and is          
# Copyright (C) Michigan State University, 2014. It is licensed under           
# the three-clause BSD license; see doc/LICENSE.txt.                            
# Contact: khmer-project@idyll.org                                              
#    
```
We should update copyright year to 2014 in all files; point to 'LICENSE' instead of doc/LICENSE.txt; and make sure that each file has this, and that they're not docstrings.

**Comment by dotsdl** [comment]
Working on this.

**Comment by dotsdl** [comment]
Is this still open? Looks like it might be resolved.

**Comment by scottsievert** [comment]
Ditto -- looks resolved but still open.

**Comment by Magentashades** [comment]
Hello, looks like some of the copyright headers are still out of date. I'm working on fixing them.

**Comment by scottsievert** [comment]
I fixed most of them in PR #928... I almost have the merge complete but finish this IPython project first.

**Comment by Magentashades** [comment]
Hi Scott, it looks like you changed the license but not the end date on the
copyright line. I'm just doing the latter which is changing the date ranges
20??-2014 to 20??-2015 if there was a commit in 2015.

**Comment by scottsievert** [comment]
Cool! So are you/have you managed the issues with LICENSE as well?

**Comment by Magentashades** [comment]
I've been leaving that alone, hoping my stuff will merge nicely with yours.

**Comment by mr-c** [comment]
Fixed by #928
